prow/config/tide_test.go

```diff
@@ -1095,7 +1097,7 @@ func TestOrgRepoMatchMergeMethod(t *testing.T) {
 		test := test
 		t.Run(test.name, func(t *testing.T) {
 			actual := test.config.OrgRepoBranchMergeMethod(OrgRepo{Org: test.org, Repo: test.repo}, test.branch)
-			if actual != test.expected {
+			if actual != test.expected && actual != test.alternativeExpected {
```

Comment:
without knowing anything about these tests or code this looks like very aleatory behavior
why should it have alternatives modes and not be more determinisic?

Reply:
yeah. I think adding ordering would be the right fix. Trying to deflake

Reply:
doesn't this mean the actual implementation behavior is non-deterministic? seems like we need to fix that.

Reply:
fixed properly now